src/services/basicValidationMapping.js

```diff
@@ -0,0 +1,11 @@
+/** Mapping between basicValidationType and the basic validation login
+ * @param {Object} event - event triggered when a character is typed
+ */
```

Comment:
I think this was mentioned in a previous PR too, but there is a slight correction in how this is done!
```suggestion
/** 
  * Mapping between basicValidationType and the basic validation login
  * @param {Object} event - event triggered when a character is typed
  */
```